crates/aptos/src/op/key.rs

```diff
@@ -94,6 +105,56 @@ impl CliCommand<HashMap<AccountAddress, Peer>> for ExtractPeer {
     }
 }
 
+#[derive(Debug, Default, Parser)]
+pub struct NetworkKeyInputOptions {
+    /// x25519 Private key input file name
+    #[clap(long, group = "network_key_input", parse(from_os_str))]
+    private_network_key_file: Option<PathBuf>,
+
+    /// x25519 Private key encoded in a type as shown in `encoding`
+    #[clap(long, group = "network_key_input")]
+    private_network_key: Option<String>,
```

Comment:
Could you use 'conflicts_with' to ensure that only one is given? https://docs.rs/clap/1.1.2/clap/struct.Arg.html#method.conflicts_with

Reply:
Let's make a github issue for this.  There are more that probably need to handle this as well.